src/lesson_4/script.js

```diff
@@ -0,0 +1,97 @@
+let sum = 0.1 + 0.2;
+console.log( sum.toFixed(2) );
+
+const userLogin = prompt("укажите свой логин");
```

Comment:
Try to use one language in the code, here your messages are in russian, but below massages in Ukrainian